### PR TITLE
Restructure tsx files

### PR DESCRIPTION
**Pull request by ganeshpatro321**
Lets use this convention! Declaring interfaces and then using them. It improves readability and avoid confusion. Follow this till eslint and prettier is configures properly

### CODE REVIEW

File: src/components/layouts/Submenu.tsx [hunk 1]
```diff
@@ -1,16 +1,33 @@
...
+}
+
+interface SubmenuStateTypes {
+  module: string,
+  urlSlot: string,
+  methodSlot: string,
+  routeSlot: string,
+  paramSlot: string
+}
+
```
Comment (Harkishen-Singh) [inline src/components/layouts/Submenu.tsx]: This is a good approach. Please implement the same for other modules like `Benchmarks.tsx`, `Ping.tsx`.

File: src/components/layouts/Submenu.tsx [hunk 2]
```diff
@@ -1,16 +1,33 @@
...
+  constructor(props: SubmenuPropsTypes) {
+    super(props);
+
+    this.state = {
+      module: '',
+      urlSlot: '',
+      methodSlot: '',
+      routeSlot: '',
+      paramSlot: ''
+    }
```
Comment (Harkishen-Singh) [inline src/components/layouts/Submenu.tsx]: Please do this inside a constructor.

Comment (ganeshpatro321) [reply src/components/layouts/Submenu.tsx]: It is done inside the constructor.

File: src/components/layouts/Submenu.tsx (hunk 1 above)
Comment (ganeshpatro321) [reply src/components/layouts/Submenu.tsx]: Okay, will do and push

Comment (ganeshpatro321) [reply src/components/layouts/Submenu.tsx]: Done

**Comment by ganeshpatro321** [comment]
Updated and clean!

**Pull request merged**